crystal_functions/file_readwrite.py

```diff
@@ -734,6 +734,226 @@ def get_config_analysis(self):
         self.atom_type2 = atom_type2
         return [self.atom_type1, self.atom_type2]
 
+    """
```

Comment:
Add a check that the output file is a freq calculation. Maybe have that as an attribute in the read_cry_output function like: self.is_freq = True if some freq specific keyword is in the output.